src/Publications/PublicationsAzureFunctions/Startup.cs

```diff
@@ -20,25 +20,29 @@ namespace PublicationsAzureFunctions
     class Startup : FunctionsStartup
     {
         private IConfiguration _configuration;
+        
         public override void Configure(IFunctionsHostBuilder builder)
         {
-            var config = builder.Services.BuildServiceProvider()
+            _configuration = builder.Services.BuildServiceProvider()
```

Comment:
for this, can we build only the configuration early, rather than the entire service tree?

```suggestion
            _configuration = builder.ConfigurationBuilder.Build();
```

Reply:
`IFunctionsHostBuilder` only exposes the Services property - maybe another way to do this?

Reply:
yeah, my bad. misread the docs